Traceback (most recent call last):
  File "conv_net_sentence.py", line 14, in <module>
    import theano
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/__init__.py", line 110, in <module>
    from theano.compile import (
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/compile/__init__.py", line 12, in <module>
    from theano.compile.mode import *
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/compile/mode.py", line 163, in <module>
    0, 'fast_run', 'fast_compile', 'merge')
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/gof/optdb.py", line 323, in register
    super(SequenceDB, self).register(name, obj, *tags)
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/gof/optdb.py", line 71, in register
    self.add_tags(name, *tags)
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/gof/optdb.py", line 76, in add_tags
    obj = obj.copy().pop()
  File "/Volumes/Transcend/opt/anaconda3/envs/theano/lib/python2.7/site-packages/theano/misc/ordered_set.py", line 171, in pop
    raise KeyError('set is empty')
KeyError: 'set is empty'
